src/SFA.DAS.ApprenticeFeedback.Domain/Models/TrainingProvider.cs

```diff
@@ -32,6 +33,32 @@ public class TrainingProvider
                 FeedbackEligibility = source.FeedbackEligibility,
                 Status = source.Status,
             };
+
+            if (FeedbackEligibilityStatus.Deny_HasGivenFeedbackRecently == trainingProvider.FeedbackEligibility)
```

Comment:
Would this section be better off as a Switch?

Switch(trainingProvider.FeedbackEligiblity)

case: HasGivenFeedbackRecently
......
break;

case Too late after passing
case too late after pausing
case too late after withdrawing
  TimeWindow = TimeSpan.FromDays(appSettings.FinalAllowedPeriodDays) ?
  
  default:
  break;